### PR TITLE
Markdown list container

### PR DESCRIPTION
**Pull request by Zilula**
Created a title selector that returns an array of titles from out markdown files. Additionally, we created a new test to check the functionality of this selector.

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import Document from './markdown/Document';
 
+
```
Comment (mehtaphysical) [inline src/components/App.js]: Why?

File: src/selectors/getTitlesSelector.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export const getTitles = state => {
```
Comment (mehtaphysical) [inline src/selectors/getTitlesSelector.js]: This can live in your `markdownListSelectors.js` file

**Comment by mehtaphysical** [comment]
looks good. just a few comments

**Review by MariahAdams — APPROVED** [review]
Green light to merge!

**Pull request merged**